### PR TITLE
issue #25 - adjusting pom to remove classifier

### PR DESCRIPTION
**Pull request by bfitzpat**
Signed-off-by: Brian Fitzpatrick <bfitzpat@redhat.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -227,6 +226,31 @@
...
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
```
Comment (valdar) [inline pom.xml]: What is the purpose of adding this profile and plugin?

Comment (bfitzpat) [reply pom.xml]: I was just following what had been done in https://github.com/snowdrop/spring-boot-http-booster/commit/1eac1c2 -- if we don't need it, I can definitely remove it and clean that up.

Comment (bfitzpat) [reply pom.xml]: Removed the extra profile

Comment (apupier) [reply pom.xml]: as maven-failsafe-plugin wasn't used before seems that we effectively don't need it. I even prefer like that :-)

**Review by apupier — COMMENTED** [review]
I tested with the "Running the booster on a single-node OpenShift cluster" readme steps and "Running the booster standalone on your machine" and it is working fine 

for "Running the booster on OpenShift Online", I don't know how to test it yet but i think that it is the same than single-node OpenShift cluster as the s2i build is triggered too.
@valdar do you know how to test it easily? if i go to fuse online it will use the version already published but not the PR one.

File: pom.xml (hunk 1 above)
Comment (valdar) [reply pom.xml]: Cool, so I will merge it.

**Pull request merged**